SOCI 415 Network Analysis Notebooks, Dataset + Media

Alexr951 opened the pull request:
Ready to be merged

Alexr951 committed:
Removed old placeholder files and added .gitattributes for .db and .dta

Alexr951 committed:
Added new files

Alexr951 committed:
Changed file paths

Alexr951 committed:
Added author name and updated dates

NathanPalaiologos reviewed (commented):
Overall, I think the existing notebooks here are already very complete. However, there are some formatting issues and grammatical errors that may need to be corrected.

Alexr951 committed:
Changes for notebooks

- Nathan's Edits
- Added mention of the ego-network building website
- Irene's name on the first notebook
- Changes for CBDB (remove second run at dynasties and more smaller changes)

Alexr951 committed:
Update index_SOCI415.qmd

Alexr951 committed:
Merge branch 'main' into SOCI_415_Pull_Request

Alexr951 committed:
Update .gitattributes

Alexr951 committed:
Delete index_SOCI415.qmd

Removed extra index file which must have been broken in the merge

Alexr951 committed:
Changes for Soci

Alexr951 committed:
Removed dataset we could not keep and added download instructions

Alexr951 committed:
Merge branch 'main' into SOCI_415_Pull_Request

Alexr951 committed:
Updates for notebooks

Alexr951 committed:
Added more details in preparation for SOCI 415 and future ProgrammingHistorian/Similar

- Added lots of descriptions
- Fixed some grammatical errors
- Added more historical context + conclusion
- Made some code + code output cleaner

Alexr951 committed:
Updated dates

Alexr951 committed:
Final spelling Changes
